api-reference/50 Common/Object Structures/PdfExportDataGridProps/indent.md

```diff
@@ -5,7 +5,7 @@ default: 0
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies the value of the horizontal gap between data cells and groups and headers. Uses the measure units which are specified in the constructor for the [jsPDFDocument](/api-reference/50%20Common/Object%20Structures/PdfExportDataGridProps/jsPDFDocument.md '/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#jsPDFDocument') object.
```

Comment:
Specifies the value of the horizontal gap between data cells and groups and headers. Uses the measure units which are specified in the constructor of the [jsPDFDocument](/api-reference/50%20Common/Object%20Structures/PdfExportDataGridProps/jsPDFDocument.md '/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#jsPDFDocument') object.